autotst/job/job.py

```diff
@@ -269,15 +271,11 @@ def submit_conformer(self, conformer, restart=False):
 
         self.calculator.conformer = conformer
         ase_calculator = self.calculator.get_conformer_calc()
```

Comment:
I think we should pass the `ase_calculator` to the `submit_conformer` function.  That way, we only need one function to submit jobs, instead of separate `submit_transitionstate` and `submit_rotor` functions.  We could discuss this on future PR.

Reply:
Okay, I'm a bit confused with what you mean here, but I'm down to discuss this in person on Monday!